roles/common/tasks/rhel-entitlements.yml

```diff
@@ -5,17 +5,19 @@
   set_fact:
     entitlements_path: "{{ secrets_path | mandatory }}/entitlements.yml"
 
-- name: Check for existence of entitlements_path
-  stat:
-    path: "{{ entitlements_path }}"
-  register: entitlements
-
 - name: Include Red Hat encrypted variables.
-  include_vars: "{{ entitlements_path }}"
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ entitlements_path }}"
+    - empty.yml
   no_log: true
   tags:
     - vars
-  when: entitlements.stat.exists
+  register: entitlements
```

Comment:
oh, good catch! I had been during development but found a better way to implement the logic I wanted. I'll remove this line.